static/images/diagrams/image_workflow.txt

```diff
@@ -0,0 +1,40 @@
+# Use http://sequencediagram.org/ to generate image
+
+title Image-based provisioning with Puppet
+
+participant User
+participant "Provisioned Host" as Host
+participant "Foreman" as Foreman
+participant "Foreman Proxy" as Proxy
+participant "Hypervisor" as HVM
+participant DHCP
+participant DNS
+participant "Puppet master" as Puppet
+
+linear
+autonumber
+
```

Comment:
I'd put your `note over Foreman:has an Image with credentials defined` line here, and also say that the hypervisor has an image with those credentials baked in.